### PR TITLE
[Toezicht] Adding missing betrokken besturen Kerkfabriek St.-Pieter van Puurs-Sint-Amands

### PR DESCRIPTION
**Pull request by Poltergeistz**
DL-4815 

Adding Kerkfabriek St.-Pieter van Puurs-Sint-Amands migration.

### CODE REVIEW

File: config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.ttl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
+
+http://mu.semte.ch/graphs/public
```
Comment (Windvis) [inline config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.ttl:3]: This should be a .graph file I think?

File: config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
+
+<http://data.lblod.info/id/bestuurseenheden/c8f33e80-6f19-4ba1-a758-9d85f42d28d4> <http://data.lblod.info/vocabularies/erediensten/betrokkenBestuur> <http://data.lblod.info/id/betrokkenLokaleBesturen/635237108DE5818A7C9F41D1> .
```
Comment (Windvis) [inline config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql:3]: I think this is a .ttl file?

File: config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
```
Comment (claire-lovisa) [inline config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql:1]: This one should be a `.graph` file no ?

Comment (Poltergeistz) [reply config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.sparql:1]: Indeed, I don't know why I was still convinced to put .sparql instead, I'll change that in other PRs as well. :man_facepalming:

File: config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.graph [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
```
Comment (claire-lovisa) [inline config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.graph:1]: I just tested and it seems that comments are not allowed in .graph files, was the migration successful for you ?

**Review by claire-lovisa — CHANGES_REQUESTED** [review]

File: config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.graph (hunk 4 above)
Comment (Poltergeistz) [reply config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.graph]: Ah I didn't know that, sorry, I've removed the comment, it should work for you.

Besluit over budget(wijziging) eredienstbestuur
Screenshot 2023-02-22 at 09-21-38 Loket voor lokale besturen

Besluit over meerjarenplan(aanpassing) eredienstbestuur
Screenshot 2023-02-22 at 09-22-04 Loket voor lokale besturen

**Pull request merged**